vaas/vaas/settings/test.py

```diff
@@ -10,8 +10,4 @@
     }
 }
 
-TEST_RUNNER = 'django_nose.NoseTestSuiteRunner'
-
-NOSE_ARGS = [
-    '--exclude-dir=' + os.path.abspath(os.path.dirname(__file__) + '/'), '--with-xunit'
-]
+TEST_RUNNER = 'django.test.runner.DiscoverRunner'
```

Comment:
Hmm, maybe this gives us an answer that we should use a standard supported testing library by Django :)